### PR TITLE
Fix for Nav - mobile scroll for Firefox

### PR DESCRIPTION
**Pull request by javalkar-varun**
### Summary
When the screen is made small both vertically and horizontally, and there are lots of links (like the patient portal example) 2 links at the bottom don't show

Before:
before_firefox_scroll

After:
firefox_scroll

Thanks for contributing to Terra. 
@cerner/terra #

### CODE REVIEW

**Comment by mhemesath** [comment]
A simpler fix is to just ensure there is a bottom padding of the nav container equal to the height of the profile menu.

File: packages/terra-consumer-nav/src/Nav.scss [hunk 1]
```diff
@@ -42,15 +42,19 @@
     user-select: none;
     width: var(--terra-consumer-nav-width, 320px);
     z-index: 2;
-
     @media screen and (max-width: $terra-tiny-breakpoint) {
       width: 100vw;
     }
 
     @media screen and (max-width: $terra-medium-breakpoint) {
+      height: calc(100% - 60px) !important;
```
Comment (mhemesath) [inline packages/terra-consumer-nav/src/Nav.scss]: why important and where is 60px coming from?

File: packages/terra-consumer-nav/src/Nav.scss [hunk 2]
```diff
@@ -42,15 +42,19 @@
...
       overflow: auto;
       padding-top: var(--terra-consumer--mobile-nav-padding-top, 0);
       position: absolute;
+      /*stylelint-disable function-url-quotes*/
+      @-moz-document url-prefix() {
+        min-height: max-content;
```
Comment (mhemesath) [inline packages/terra-consumer-nav/src/Nav.scss]: are you sure this is needed? Also according to MDN that property value isn't supported in firefox. https://developer.mozilla.org/en-US/docs/Web/CSS/min-height

File: packages/terra-consumer-nav/src/Nav.scss (hunk 1 above)
Comment (mhemesath) [reply packages/terra-consumer-nav/src/Nav.scss]: If the 60px is the profile height, i'm confused why line 39 is not fixing this problem as that is in place to address this issue.

File: packages/terra-consumer-nav/src/Nav.scss [hunk 3]
```diff
@@ -42,15 +42,19 @@
...
       width: 100vw;
     }
 
     @media screen and (max-width: $terra-medium-breakpoint) {
+      height: calc(100% - 60px) !important;
       overflow: auto;
       padding-top: var(--terra-consumer--mobile-nav-padding-top, 0);
       position: absolute;
+      /*stylelint-disable function-url-quotes*/
+      @-moz-document url-prefix() {
```
Comment (mhemesath) [inline packages/terra-consumer-nav/src/Nav.scss]: I don't think you need this at all... just leave `min-height: max-content;`

File: packages/terra-consumer-nav/src/Nav.scss (hunk 1 above)
Comment (mhemesath) [reply packages/terra-consumer-nav/src/Nav.scss]: can this just be applied on line 35? Also shouldn't need important

**Comment by zhongyn** [comment]
@javalkar-varun can you try the padding solution suggested by mike?

**Comment by zhongyn** [comment]
@javalkar-varun I think the root cause is line 53 `position: absolute`. Switching it to `relative` fixed the scroll issue. Tested it in chrome/firefox/ie.

File: packages/terra-consumer-nav/src/Nav.scss [hunk 4]
```diff
@@ -42,15 +42,14 @@
...
-
     @media screen and (max-width: $terra-tiny-breakpoint) {
       width: 100vw;
     }
 
     @media screen and (max-width: $terra-medium-breakpoint) {
       overflow: auto;
       padding-top: var(--terra-consumer--mobile-nav-padding-top, 0);
-      position: absolute;
+      position: relative;
```
Comment (zhongyn) [inline packages/terra-consumer-nav/src/Nav.scss]: can you move it outside of the media query. It should apply to both desktop and mobile

File: packages/terra-consumer-nav/src/Nav.scss [hunk 5]
```diff
@@ -42,15 +42,14 @@
     user-select: none;
     width: var(--terra-consumer-nav-width, 320px);
     z-index: 2;
-
     @media screen and (max-width: $terra-tiny-breakpoint) {
       width: 100vw;
     }
 
     @media screen and (max-width: $terra-medium-breakpoint) {
       overflow: auto;
```
Comment (zhongyn) [inline packages/terra-consumer-nav/src/Nav.scss]: i think this is no needed anymore

Comment (javalkar-varun) [reply packages/terra-consumer-nav/src/Nav.scss]: yes, removed.

File: packages/terra-consumer-nav/src/Nav.scss (hunk 4 above)
Comment (javalkar-varun) [reply packages/terra-consumer-nav/src/Nav.scss]: yes, moved. Thanks for the input.

File: packages/terra-consumer-nav/src/Nav.scss [hunk 6]
```diff
@@ -38,19 +38,18 @@
     overflow-y: auto;
     padding-bottom: 60px;
     padding-top: var(--terra-consumer-nav-padding-top, 25px);
+    position: relative;
```
Comment (mhemesath) [inline packages/terra-consumer-nav/src/Nav.scss]: interesting, you just needed to add relative positioning? Does this fix it in IE too?

Comment (javalkar-varun) [reply packages/terra-consumer-nav/src/Nav.scss]: Yes, win10 Edge and IE
image

Comment (javalkar-varun) [reply packages/terra-consumer-nav/src/Nav.scss]: yes, win10 Edge and IE. I think height: 100% which is normal position but when position is set to relative, nav behaves w.r.t to normal position.

**Pull request merged**